src/LttngHelpers.c

```diff
@@ -1414,7 +1414,7 @@ int lttngh_EventProbe(
                     {
                         uint16_t cbUtf8Written;
 #if lttngh_UST_RING_BUFFER_NATURAL_ALIGN
-                        lib_ring_buffer_align_ctx(&bufferContext, pDataDesc[i].Alignment);
+                        lttng_ust_ring_buffer_align_ctx(&bufferContext, pDataDesc[i].Alignment);
```

Comment:
I don't think `lttng_ust_ring_buffer_align_ctx` is available on 2.08. This may need version IFDEFs.

Reply:
I think we can actually just update the docs here to claim no support on LTTNG < 2.10.  Ubuntu 16.04 was the only thing we were supporting (and didn't wind up ever actually using), and has now even aged out of LTS and is an unsupported operating system.

Reply:
Ok, but when was the name changed? Was it renamed for 2.10?

Reply:
How do I check if that is/isn't available on 2.08? checked online doc and neither are listed.

Reply:
LttngHelpers runs at a layer below the officially-supported LTTNG API, so it uses stuff that isn't listed in the docs.

I would download the lttng-ust-2.XX.tar.bz2 source files, extract them, and see what version stops having `lib_ring_buffer_align_ctx` and starts having `lttng_ust_ring_buffer_align_ctx`. My guess is that it gets renamed at version 2.13.

We have a section at the top of this file that has IFDEFs for various versions. You would probably want to add a macro to that section. One macro would be defined to `lttng_ust_ring_buffer_align_ctx` and the other would be defined to `lib_ring_buffer_align_ctx`.

This change needs to be validated on both the lowest version of LTTNG we support and the latest. The problem is that this code is only exercised when `lttngh_UST_RING_BUFFER_NATURAL_ALIGN` is set, so it is IFDEFed-out during the CI build.

Reply:
I checked the `lttng_ust_ring_buffer_align` and `lttng_ust_ring_buffer_align_ctx` only show up in 2.13. in 2.12 and below we have `lib_ring_buffer_align` and `lib_ring_buffer_align_ctx`. 
Additionally I think the `lttng_ust_ring_buffer_align_ctx` is not exposed in 2.13 and we dont call in in 2.13 we only call `lib_ring_buffer_align_ctx` when the version is less than 2.13 and `lttngh_UST_RING_BUFFER_NATURAL_ALIGN` is set to 1